app.py

```diff
@@ -44,8 +44,11 @@ def create_app(env: str = "test", context: str = "main") -> Flask:
 
     # Health check
     @app.route("/health/")
-    def health_check():
-        return {"status": "ok"}
+    def health_check() -> dict:
+        try:
+            return {"status": "ok"}
+        except Exception as exp_err:
+            return {"Error": str(exp_err)}
```

Comment:
No need to add this in try except. Keep it as it is.

Reply:
ok